.github/ISSUE_TEMPLATE/server_error_template.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Server error
+about: Report a production bug from the server logs
+---
+<!--
+  - Before filing a new issue, please do a quick search to check that it hasn't
+  - already been filed on the [issue tracker](https://github.com/oppia/oppia/issues)._
+  -->
+
+This error occurred recently in production
+
+```
+Add error logs here.
+```
+
+**Additional context** Add details from the logs, like the page the error occurred on.
```

Comment:
Maybe add some notes to ensure anonymization

Reply:
Done